src/js/components/Graphs/Timeline/timeline.jsx

```diff
@@ -14,31 +14,25 @@
  * limitations under the License.
  */
 
-import React, { useEffect } from 'react';
+import React, { useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
-import { constructTimeline } from '../../helpers/timeline/declarationHelper';
-import { drawTimeline } from '../../helpers/timeline/drawHelper';
-import { TimelineContainer, Timeline } from './timeline.style';
-
-import 'vis-timeline/dist/vis-timeline-graph2d.min.css';
-
-const selectors = {
-  TIMELINE: '.timelineContainer .timeline',
-};
+import { constructTimeline } from '../../../helpers/timeline/declarationHelper';
+import { drawTimeline } from '../../../helpers/timeline/drawHelper';
+import { Timeline } from './timeline.style';
 
 const TimelineComponent = ({
   graphJson,
 }) => {
+  const timeline = useRef(null);
```

Comment:
`null`? It looks tricky here.

Reply:
We can introduce a constant here (e.g`self`).

Reply:
its not tricky. look at https://pl.reactjs.org/docs/hooks-reference.html#useref

Reply:
It's not self, it is creating empty ref that will be attached to dom node after it will be rendered